docs/release_notes/sat_2.2_release_notes.md

```diff
@@ -68,10 +70,10 @@ To resolve, run `sat` in another directory.
 ### Duplicate Mount Error when Running `sat` in Configuration Directory
 
 `sat` commands will not work if the current directory is `~/.config/sat`.
-For example:
 
-```screen
-ncn-m001:~/.config/sat # sat --help
+(`ncn-m001:~/.config/sat #`) Here is an example output after running `sat --help`:
```

Comment:
Again, this one's kinda tricky because you are trying to show that the working directory is `~/.config/sat`, but that's not part of our command-prompt conventions, so I'd still omit that part.

```suggestion
(`ncn-m001#`) Here is an example output after running `sat --help`:
```

Reply:
Updated in next commit.